packages/@aws-cdk/aws-events-targets/package.json

```diff
@@ -106,6 +106,7 @@
     "@aws-cdk/aws-sns-subscriptions": "1.23.0",
     "@aws-cdk/aws-sqs": "1.23.0",
     "@aws-cdk/aws-stepfunctions": "1.23.0",
+    "@aws-cdk/aws-batch": "1.23.0",
```

Comment:
You need to merge from `master` (which will lead to lots of conflicts). These days we version using the symbolic version `"0.0.0"` to avoid constant merge conflicts when releasing.